### PR TITLE
Wind: cleanup of texture creation code

### PR DESCRIPTION
**Pull request by 1chandu**
- Remove lot of redundant/unused props in texture creation path.

### CODE REVIEW

File: examples/wind/src/layers/particle-layer/particle-layer.js [hunk 1]
```diff
@@ -332,24 +334,21 @@ export default class ParticleLayer extends Layer {
   }
 
   createTexture(gl, opt) {
```
Comment (gnavvy) [inline examples/wind/src/layers/particle-layer/particle-layer.js]: `createTexture(gl, opt = {}) {`
so we don't need the 2nd param in line 51 and 52

File: examples/wind/src/layers/wind-layer/wind-layer.js [hunk 2]
```diff
@@ -166,24 +165,21 @@ export default class WindLayer extends Layer {
   }
 
   createTexture(gl, opt) {
```
Comment (gnavvy) [inline examples/wind/src/layers/wind-layer/wind-layer.js]: same as above, let opt = {} by default.
but how do we control the texture size without passing down the width and height?

File: examples/wind/src/layers/particle-layer/particle-layer.js (hunk 1 above)
Comment (1chandu) [inline examples/wind/src/layers/particle-layer/particle-layer.js]: Yeah, if you look at the flow, they are never passed down to Texture2D constructor.

File: examples/wind/src/layers/wind-layer/wind-layer.js (hunk 2 above)
Comment (1chandu) [inline examples/wind/src/layers/wind-layer/wind-layer.js]: Same as above.

File: examples/wind/src/layers/particle-layer/particle-layer.js (hunk 1 above)
Comment (shaojingli) [inline examples/wind/src/layers/particle-layer/particle-layer.js]: Can we put this as comment in the code?

**Pull request merged**